### DISCUSSION TITLE
Github copilot plugin update error, JetBrains Rider

### DISCUSSION THREAD

**Discussion by Firuz2006** [community/community#143030] [Copilot Conversations] [answered]
### Select Topic Area

Product Feedback

### Body

![image](https://github.com/user-attachments/assets/47cd9a2d-029c-4b44-aab2-37ec122a33b3)

**Comment by dougwaldron** [comment, marked as answer]
I have the same problem. I can uninstall and reinstall without issue, but I can't directly update.

**Comment by jiec-msft** [comment]
@Firuz2006 Thanks for reporting the issue. We released a stable version [1.5.30-xxx](https://plugins.jetbrains.com/plugin/17718-github-copilot/versions) with the fix. But it will take effect after it's installed. So, a work around as following, and sorry for any inconvenience.

1. First disable the plugin & restart IDE
1. Uninstall copilot plugin & restart IDE
1. Install the latest version of GitHub Copilot plugin.
